app/assets/stylesheets/components/_buttons.scss

```diff
@@ -0,0 +1,11 @@
+button,
+input[type="button"],
+input[type="reset"],
+input[type="submit"],
+.button-link {
+  margin-top: 15px;
```

Comment:
All buttons have a top margin? Maybe a style change would be made somewhere else to do what this spacing is doing.